snap/helpers/app.ini

```diff
@@ -0,0 +1,69 @@
+APP_NAME    = Gitea: Go Git Service
+RUN_USER    = root
```

Comment:
Should gitea really run as root? Or do I misinterpret this variable?

Reply:
It's not a misinterpretation.
Snap packages right now all 'run as root' for daemons.

The security part here is defined via apparmor confinement. With the plugs in snapcraft.yaml the root user is only allowed to access the snap directories and network access and port binding.

Reply:
That's all right then.